cellprofiler/modules/unmixcolors.py

```diff
@@ -207,65 +209,88 @@ def add_image(self, can_remove=True):
-            <tr><td>%(CHOICE_AZOCARMINE)s</td><td bgcolor="%(COLOR_AZOCARMINE)s">&nbsp;</td><td>Plasma</td></tr>
-            <tr><td>%(CHOICE_DAB)s</td><td bgcolor="%(COLOR_DAB)s">&nbsp;</td><td>Peroxisomes, mitochondria</td></tr>
-            <tr><td>%(CHOICE_EOSIN)s</td><td bgcolor="%(COLOR_EOSIN)s">&nbsp;</td><td>Elastic, collagen and reticular fibers</td></tr>
-            <tr><td>%(CHOICE_FAST_RED)s</td><td bgcolor="%(COLOR_FAST_RED)s">&nbsp;</td><td>Nuclei</td></tr>
-            <tr><td>%(CHOICE_FAST_BLUE)s</td><td bgcolor="%(COLOR_FAST_BLUE)s">&nbsp;</td><td>Myelin fibers</td></tr>
-            <tr><td>%(CHOICE_FEULGEN)s</td><td bgcolor="%(COLOR_FEULGEN)s">&nbsp;</td><td>DNA</td></tr>
-            <tr><td>%(CHOICE_HEMATOXYLIN)s</td><td bgcolor="%(COLOR_HEMATOXYLIN)s">&nbsp;</td><td>Nucleic acids, endoplasmic reticulum</td></tr>
-            <tr><td>%(CHOICE_HEMATOXYLIN_AND_PAS)s</td><td bgcolor="%(COLOR_HEMATOXYLIN_AND_PAS)s">&nbsp;</td><td>Nucleus (stained with both Hematoxylin and PAS)</td></tr>
-            <tr><td>%(CHOICE_METHYL_BLUE)s</td><td bgcolor="%(COLOR_METHYL_BLUE)s">&nbsp;</td><td>Collagen</td></tr>
-            <tr><td>%(CHOICE_METHYL_GREEN)s</td><td bgcolor="%(COLOR_METHYL_GREEN)s">&nbsp;</td><td>Chromatin</td></tr>
-            <tr><td>%(CHOICE_METHYLENE_BLUE)s</td><td bgcolor="%(COLOR_METHYLENE_BLUE)s">&nbsp;</td><td>Nuclei</td></tr>
-            <tr><td>%(CHOICE_ORANGE_G)s</td><td bgcolor="%(COLOR_ORANGE_G)s">&nbsp;</td><td>Erythrocytes, pancreas, pituitary</td></tr>
-            <tr><td>%(CHOICE_PAS)s</td><td bgcolor="%(COLOR_PAS)s">&nbsp;</td><td>Glycogen, carbohydrates</td></tr>
-            <tr><td>%(CHOICE_PONCEAU_FUCHSIN)s</td><td bgcolor="%(COLOR_PONCEAU_FUCHSIN)s">&nbsp;</td><td>Red counterstain for Masson's trichrome</td></tr>
-            </table>
-            <br>
-            (Information taken from <a href="http://en.wikipedia.org/wiki/Histology#Staining">here</a>,
-            <a href="http://en.wikipedia.org/wiki/Staining">here</a>, and
-            <a href="http://stainsfile.info">here</a>.)
-            <br>
-            You can choose <i>%(CHOICE_CUSTOM)s</i> and enter your custom
-            values for the absorbance (or use the estimator to determine values
-            from a single-stain image).
-            """ % globals()))
+                "Stain", choices=choices, doc="""\
+Use this setting to choose the absorbance values for a particular stain.
+
+The stains are:
++----------------------------------------------+---------+---------------------------------------------------+
+| Stain                                        | Color   | Specific to                                       |
++==============================================+=========+===================================================+
+| %(CHOICE_AEC)s (3-Amino-9-ethylcarbazole)    |         | Peroxidase                                        |
```

Comment:
I don't think there's a way to change the cell color in a table using reStructuredText. 

I see two options: 
1. Generate color thumbnails and embed the images in the color column, or
2) Remove the color column